frontend/styles/theme/lightThemeOptions.ts

```diff
@@ -2,6 +2,11 @@ import { ThemeOptions } from '@mui/material/styles';
 import { alpha } from "@mui/material";
 
 const lightThemeOptions: ThemeOptions = {
+  typography: {
+    h1:{
+      fontSize: '3rem',
```

Comment:
On pourrait un jour se fier à un genre de guide line pour toute les fontSize. Ex: [FontSize Guide](https://www.impactplus.com/blog/best-font-size-for-website#:~:text=Although%20there's%20no%20official%20ADA,font%20for%20the%20body%20text.)

Reply:
Yes ça pourra être changé dans le futur. Bonne référence!